src/lib/jsonParseV2/parseContextAttributeCompoundValue.cpp

```diff
@@ -124,13 +125,13 @@ std::string parseContextAttributeCompoundValue
   }
   else if (node->IsArray())
   {
-    int counter  = 0;
+    short counter  = 0;
```

Comment:
Discussed by skype: revert to int usage and make snprintf buffers larger.

Reply:
Fixed in 3cb7510